call/index.js

```diff
@@ -0,0 +1,14 @@
+const passwordForm = document.getElementById('password-form');
+const submitBtn = document.getElementById('submit-btn');
+
+submitBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    const passwordInput = document.getElementById('password').value.trim();
+
+    if (passwordInput === 'thehackstrikesback') {
```

Comment:
You never want to commit a password or secret to version control. It is not secure. Instead, it can be set as an environment variable.